build/gen-odf-package.sh

```diff
@@ -27,7 +27,7 @@ head -n ${n} ${MANIFESTS}/${CSV_NAME} > ${temp_csv}
 cat >> ${temp_csv} << EOF
   relatedImages:
   - image: ${CORE_IMAGE}
-    name: noboaa-core
```

Comment:
it took me some time to understand what is the difference here :) 
In general, I think that it's better to make these kind of changes in a different PR, so it will be easier to backport if needed.
In this specific case, it shouldn't be a porblem, but worth checking 
https://docs.openshift.com/container-platform/4.9/operators/operator_sdk/osdk-generating-csvs.html#olm-enabling-operator-for-restricted-network_osdk-generating-csvs